modules/combined/tests/phase_field_fracture/crack2d.i

```diff
@@ -48,18 +48,20 @@
     G0_var = 'G0_pos'
     dG0_dstrain_var = 'dG0_pos_dstrain'
   [../]
+  [./DynamicTensorMechanics]
```

Comment:
Well either one works. I had used DynamicTensorMechanics to check if it works with both TensorMechanics and DynamicTensorMechanics. I can change it back to TensorMechanics if needed.